ProjectSettings/GraphicsSettings.asset

```diff
@@ -39,9 +39,21 @@ GraphicsSettings:
   m_PreloadedShaders: []
   m_SpritesDefaultMaterial: {fileID: 10754, guid: 0000000000000000f000000000000000,
     type: 0}
+<<<<<<< HEAD
```

Comment:
What's the best way to revert?  I thought I had but obviously not.  D'oh!

thanks,
-pat